src/ContainerCheckpointModal.jsx

```diff
@@ -24,6 +24,8 @@ class ContainerCheckpointModal extends React.Component {
     render() {
         return (
             <Modal isOpen={this.props.selectContainerCheckpointModal}
+                   onEscapePress={() => undefined}
+                   showClose={false}
                    position="top" variant="medium"
                    onClose={this.props.handleCheckpointContainerDeleteModal}
```

Comment:
We don't need to define `onClose` then anymore. And if you drop that, then there is no need to define `onEscapePress`.

I guess we cannot disable the X button as well, if we could, that would be the best approach. But otherwise not showing the X button works as well.

Reply:
We can't - but I will open an issue to PF4 and maybe I can implement it the following week.

Reply:
That would be great, thanks! :)

Reply:
https://github.com/patternfly/patternfly-design/issues/952